libmachine/drivers/rpc/server_driver.go

```diff
@@ -108,8 +124,22 @@ func (r *RPCServerDriver) SetConfigRaw(data []byte, _ *struct{}) error {
 	return json.Unmarshal(data, &r.ActualDriver)
 }
 
-func (r *RPCServerDriver) Create(_, _ *struct{}) error {
-	return r.ActualDriver.Create()
+func trapPanic(err *error) {
+	if r := recover(); r != nil {
+		*err = fmt.Errorf("Panic in the driver: %s\n%s", r.(error), stdStacker.Stack())
+	}
+}
+
+func (r *RPCServerDriver) Create(_, _ *struct{}) (err error) {
```

Comment:
I should note for future reference that while I'm usually against named return parameters, it's actually needed here.  (otherwise the `defer`ed function can't modify the value of the parameter)